run-tests.php

```diff
@@ -122,6 +122,8 @@ function show_usage()
 HELP;
 }
 
+include('tests-util/OutputHandler.php');
```

Comment:
run-tests.php should remain a single file.

Reply:
and single it shall remain @nikic - we're sticking with single. code updated and approach going fwd will be single-file too.